plasmapy/analysis/swept_langmuir/tests/test_floating_potential.py

```diff
@@ -16,13 +17,35 @@
 )
 
 
+def test_floating_potential_namedtuple():
+    """
+    Test structure of the namedtuple used to return computed floating potential
+    data.
+    """
+
+    assert issubclass(FloatingPotentialResults, tuple)
+    assert hasattr(FloatingPotentialResults, "_fields")
+    assert FloatingPotentialResults._fields == (
+        "vf",
+        "vf_err",
+        "rsq",
+        "func",
+        "islands",
+        "indices",
+    )
+    if sys.version_info >= (3, 7):
+        # TODO: remove if clause when python 3.6 support is dropped
+        assert hasattr(FloatingPotentialResults, "_field_defaults")
+        assert FloatingPotentialResults._field_defaults == {}
```

Comment:
I would totally be okay with dropping 3.6 support for our next development release, especially following [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html).  Plus, Astropy 4.2 requires Python 3.7, so we would likely effectively be dropping Python 3.6 support in the next year anyway.

Reply:
Likewise.  I'm trying to leave these TODOs in key places so it will be easier go back and update when the py36 drop PR arrives.